addons/capetrader/capetrader.lua

```diff
@@ -0,0 +1,495 @@
+--[[Copyright © 2016, Lygre, Burntwaffle@Odin
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of CapeTrader nor the
+      names of its contributors may be used to endorse or promote products
+      derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL Lygre, Burntwaffle@Odin BE LIABLE FOR ANY
+DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.]]--
+
+_addon.name = 'CapeTrader'
+_addon.author = 'Lygre, Burntwaffle'
+_addon.version = '1.0.2'
+_addon.commands = {'capetrader', 'ct'}
+
+require('luau')
+require('pack')
+require('sets')
+require('functions')
+require('strings')
+local res = require('resources')
+local packets = require('packets')
+local augPaths = require('allAugPaths')
+local maxAugMap = require('maxAugMap')
+local extData = require('extdata')
+local jobToCapeMap = require('jobToCapeMap')
+
+local playerIndex = nil
+local gorpaID = 17797273
```

Comment:
I added some code in order to quickly check these values after a version update. If you could  have a look at the printGorpaPacketInfo() function and the windower.register_event('outgoing chunk') portion of code and maybe tell me if there is a better way to get that information that would be great. I also added some comments to the capetrader.lua file to explain how to use those functions to get the relevant information in case I am not around to patch the addon.